### ISSUE TITLE
provide SingleConsumer*ControlAwareMailbox

### ISSUE THREAD

**Issue by hepin1989** [akka/akka#24877]
make it backed by two `SingleConsumerOnly*Mailbox`

**Comment by johanandren** [comment]
I'm not sure we would want to provide this, but the API is open for writing your own mailbox, so should be able to implement one yourself if you have the need.

**Comment by hepin1989** [comment]
I do have the need :) let's keep it open.

And if we are using the Akka Scheduler to scheduling something,Then there could be nice to have a SPSCConsumerControlAwaredMailbox too.

**Comment by patriknw** [comment]
I don't think we want an explosion of number of supported mailbox implementations, but you can implement it yourself in your own project/library.
